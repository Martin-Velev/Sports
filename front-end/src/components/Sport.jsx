import React from "react";
import { Link } from "react-router-dom";

const Sport = props => {
  const { sport } = props;
  return (
    <Link to={`sports/${sport.id}`}>
        {sport.description}({sport.id})
    </Link>
  );
};

export default Sport;
