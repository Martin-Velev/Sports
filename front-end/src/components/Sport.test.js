import React from "react";
import renderer from "react-test-renderer";
import Sport from "../components/Sport";
import { BrowserRouter as Router } from "react-router-dom";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Router>
        <Sport sport={{id: 42}} />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
