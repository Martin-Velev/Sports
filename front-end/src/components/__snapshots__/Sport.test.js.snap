// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  href="/sports/42"
  onClick={[Function]}
>
  (
  42
  )
</a>
`;
